### PR TITLE
Improvements for Readme

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Improved readme
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | n/a
| Related PRs       | n/a
| How to test?      | Please indicate how to best verify that this PR is correct.
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,12 +123,12 @@ PrestaShop is a very extensible e-commerce platform, both through modules and th
...
+You can discuss about e-commerce, help other merchants and get help, and contribute to improving PrestaShop together with the PrestaShop community on [PrestaShop Slack channel][chat] or on the [the PrestaShop forums][forums].
 
 Thank you for downloading and using the PrestaShop Open Source e-commerce solution!
 
 [available-features]: https://www.prestashop.com/en/online-store-builder
-[download]: https://www.prestashop.com/en/download
+[download]: https://github.com/PrestaShop/PrestaShop/releases
 [forums]: https://www.prestashop.com/forums/
 [chat]: https://www.prestashop-project.org/slack/
 [user-doc]: https://doc.prestashop.com
```
Comment (PrestaEdit) [inline README.md]:
```suggestion
[user-doc]: https://docs.prestashop-project.org
```

File: README.md [hunk 2]
```diff
@@ -139,16 +139,16 @@ Thank you for downloading and using the PrestaShop Open Source e-commerce soluti
 [user-guide]: https://docs.prestashop-project.org/1.7-documentation/user-guide
```
Comment (PrestaEdit) [inline README.md]:
```suggestion
[user-guide]: https://docs.prestashop-project.org/v.8-documentation/v/english/user-guide
```

File: README.md [hunk 3]
```diff
@@ -139,16 +139,16 @@ Thank you for downloading and using the PrestaShop Open Source e-commerce soluti
...
 [sysadmin-guide]: https://docs.prestashop-project.org/1-6-documentation/english-documentation/system-administrator-guide
 [contributors-md]: CONTRIBUTORS.md
-[example-nginx]: https://devdocs.prestashop.com/8/basics/installation/nginx/
+[example-nginx]: https://devdocs.prestashop-project.org/8/basics/installation/nginx/
 [docker-compose]: https://docs.docker.com/compose/
-[install-guide-dev]: https://devdocs.prestashop.com/8/basics/installation/
-[system-requirements]: https://devdocs.prestashop.com/8/basics/installation/system-requirements/
+[install-guide-dev]: https://devdocs.prestashop-project.org/8/basics/installation/
+[system-requirements]: https://devdocs.prestashop-project.org/8/basics/installation/system-requirements/
 [install-guide]: https://docs.prestashop-project.org/1.7-documentation/getting-started/installing-prestashop
```
Comment (PrestaEdit) [inline README.md]:
```suggestion
[install-guide]: https://docs.prestashop-project.org/v.8-documentation/v/english/getting-started/installing-prestashop
```

**Review by nicosomb — APPROVED** [review]

**Review by 0x346e3730 — APPROVED** [review]

**Review by mflasquin — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @kpodemski

**Pull request merged**